var/spack/repos/jcsda-emc/packages/saber/CMakeLists.txt.patch

```diff
@@ -0,0 +1,11 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -42,7 +42,7 @@ find_package( atlas 0.33.0 REQUIRED COMPONENTS OMP_Fortran )
+ if( ENABLE_MKL )
+     find_package( MKL )
```

Comment:
In fact, this should rather be
```
if( ENABLE_MKL )
    find_package( MKL REQUIRED )
```
(at least that is what we decided at JCSDA - if you request a feature then it is an error if it is not available)

Reply:
Actually, we already have the change that you made in develop, and your version makes more sense than what I was thinking initially. That's because it's easier going to use Intel MKL  *or* LAPACK, but if both are missing it will abort.

Reply:
My idea doesn't work, fixing the underlying problem requires changes across several repositories.